portfolio/io.go

```diff
@@ -370,9 +371,9 @@ func ToCsvString(txs []*Tx) string {
 			tx.TradeDate.String(),
 			tx.SettlementDate.String(),
 			tx.Action.String(),
-			fmt.Sprintf("%d", tx.Shares),
-			fmt.Sprintf("%f", tx.AmountPerShare),
-			fmt.Sprintf("%f", tx.Commission),
+			tx.Shares.String(), // TODO formatting
```

Comment:
Can you elaborate? Is this doing like 2.000000 ?

Reply:
🤔  I think this was just an open question for how these need to be formatted compared to floats but considering no rounding/truncating was happening before, I think `String()` is the equivalent here and fine.